scripts/pipelines/azure-devops/templates/eks/eks-provisioning.yml.template

```
@@ -0,0 +1,70 @@
+trigger:
+- none
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+- group: AWS_Credentials
+- name: terraformWorkingDirectory
+  value: '$(System.DefaultWorkingDirectory)/.terraform/eks'
+
+steps:
+
+- task: TerraformInstaller@0
+  displayName: 'Install Terraform latest'
+  inputs:
+    terraformVersion: 'latest' 
+
+- task: TerraformTaskV2@2
+  displayName: 'Terraform : init'
+  inputs:
+    provider: 'aws'
+    command: 'init'
+    workingDirectory: "$(terraformWorkingDirectory)"
+    backendServiceAWS: "$(awsServiceConnection)"
+    backendAWSBucketName: "$(s3Bucket)"
+    backendAWSKey: "$(s3KeyPath)"
+
+- task: TerraformTaskV2@2
+  displayName: 'Terraform : apply'
+  name: terraformApply
+  inputs:
+    provider: aws
+    command: apply
+    commandOptions: '-var cluster_name=$(clustername)'
+    workingDirectory: "$(terraformWorkingDirectory)"
+    environmentServiceNameAWS: "$(awsServiceConnection)"
+
+- task: Bash@3
+  displayName: 'Install NGINX Ingress controller'
+  inputs:
+    targetType: inline
+    script: |
+      aws configure set aws_access_key_id $(ACCESS_KEY)
+      aws configure set aws_secret_access_key $(SECRET_KEY)
+      aws eks update-kubeconfig --name $(clustername) --region $(REGION)
+      sh '$(System.DefaultWorkingDirectory)/.pipelines/scripts/install-ingress-controller.sh'
```

```suggestion
      $(System.DefaultWorkingDirectory)/.pipelines/scripts/install-ingress-controller.sh
```

File should be executable already. Please check.